charts/layer0_describo/templates/ingress.yaml

```diff
@@ -15,10 +15,13 @@ spec:
       secretName: {{ include "common.tlsSecretName" . }}
   {{- end }}
   rules:
-    - host: {{ .Values.global.describo.domain }}
+    - host: {{ .Values.global.rds.domain }}
       http:
         paths:
           - path: {{ .Values.ingress.path }}
+            pathType: Prefix
             backend:
-              serviceName: {{ $fullName }}
-              servicePort: http
+              service:
+                name: {{ $fullName }}
+                port:
+                  number: 80
```

Comment:
Is it possible to refer to the port by name still, i.e. `name: http`?

Reply:
Not that I can tell from the docs: https://kubernetes.io/docs/concepts/services-networking/ingress/

Reply:
As discussed in gitter and just for future reference/reviewers, it should still work according to API docs:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#servicebackendport-v1-networking-k8s-io

Reply:
I tested it and  yes it does also work with name: http